### DISCUSSION TITLE
Guru Meditation Error: Core  1 panic'ed (LoadProhibited)

### DISCUSSION THREAD

**Discussion by geekblack22** [schreibfaul1/ESP32-audioI2S#184] [Q&A]
I just suddenly started getting this error can anybody help?

Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
Core 1 register dump:
PC      : 0x40090c37  PS      : 0x00060630  A0      : 0x8009120b  A1      : 0x3ffb1e30  
A2      : 0x3f800000  A3      : 0x000493ec  A4      : 0x00000000  A5      : 0x3ffb1dc0  
A6      : 0x0bfffffc  A7      : 0x00000000  A8      : 0x00000000  A9      : 0xbbbbbbbb  
A10     : 0x3f800010  A11     : 0x3ffc46c8  A12     : 0x3f800014  A13     : 0x00000000  
A14     : 0xffffffff  A15     : 0x00000000  SAR     : 0x00000005  EXCCAUSE: 0x0000001c  
EXCVADDR: 0xbbbbbbbb  LBEG    : 0x4008ba12  LEND    : 0x4008ba1d  LCOUNT  : 0x00000000  

ELF file SHA256: 0000000000000000

Backtrace: 0x40090c37:0x3ffb1e30 0x40091208:0x3ffb1e50 0x40081f2d:0x3ffb1e70 0x40082104:0x3ffb1e90 0x40081547:0x3ffb1eb0 0x400dc1a6:0x3ffb1ed0 0x400dcb10:0x3ffb1f00 0x400d2a33:0x3ffb1f80 0x400f2802:0x3ffb1fb0 0x4008db96:0x3ffb1fd0

**Comment by Nickbuster** [comment]
Have you resolved this?
I am experiencing same issue.

**Comment by schreibfaul1** [comment]
It's a memory problem, I don't know your source code.
It might help to use a larger data partition, e.g. 'HUGE APP'

**Reply by Nickbuster** [reply]
Correct!
My ESP32 board was faulty. PSRAM issue.
I have replaced it with new one.
Now it works.
